frontend/src/theme/buttonStyles.tsx

```diff
@@ -0,0 +1,60 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme";
+import textStyles from "./textStyles";
+
+const Button: ComponentStyleConfig = {
+  baseStyle: {
+    color: "white.default",
+    width: "auto",
+    ...textStyles["button-medium"],
+  },
+  variants: {
+    primary: {
```

Comment:
the primary, secondary and tertiary buttons should have bold text. You can do this by adding this to each variant (except navbar):
```
      fontWeight: "bold",
```